### PR TITLE
Revise Welcome Wagon guide

### PR DESCRIPTION
**Pull request by ddbeck**
I went through Welcome Wagon guide, fixed some formatting issues I saw, and revised it based on my experience co-hosting in Prague.

cc: @bethaitman

### CODE REVIEW

File: docs/organizer-guide/confs/welcome-wagon.rst [hunk 1]
```diff
@@ -23,45 +23,45 @@ As a WW volunteer, you represent WTD before and during the conference to make th
 
 Your most important function as a WW volunteer is to be available for any questions or any help at any time. You are a recognizable face that can make the conference easy and fun for first-timers.
 
-Ideally, the Welcome Wagon volunteers should be locals to the city where the conference is run and have previously attended a Write the Docs conference.
+The WW volunteers have previously attended a Write the Docs conference and, ideally, are  locals to the city where the conference held.
```
Comment (plaindocs) [inline docs/organizer-guide/confs/welcome-wagon.rst]: *is* held

File: docs/organizer-guide/confs/welcome-wagon.rst [hunk 2]
```diff
@@ -72,11 +72,11 @@ Meet and Greet
...
-*	Right before the Meet and Greet starts, do a quick sweep of the Opening Reception and invite any first-time attendees to join in the Meet and Greet.
-*	When you start the Meet and Greet, introduce yourself and the Welcome Wagon events.
-*	Split up the attendees into smaller groups and have each WW volunteer lead a group: review all of the information from your notes, take questions, and have first-time attendees introduce themselves to each other.
-*	At the end of the Meet and Greet, each volunteer should walk with their group to the Opening Reception.
-*	At the Opening Reception, Welcome Wagon volunteers, if they are comfortable with it, can approach attendees who are alone to help make them feel involved at the event. A great icebreaker is, “Is this your first conference?” Even if it isn’t, it is a good way to get a conversation started. If the attendee seems comfortable not talking or being alone at the event, by all means, let them be.
+*	Right before the Meet and Greet starts, ask the keeper of the triangle to sweep the Opening Reception to invite any first-time attendees to join in the Meet and Greet.
```
Comment (plaindocs) [inline docs/organizer-guide/confs/welcome-wagon.rst]: KotT, like it

**Review by plaindocs — COMMENTED** [review]
Looks great!

I'm personally less keen on WW, what do you think about using long form throughout?

**Comment by ddbeck** [comment]
> I'm personally less keen on WW, what do you think about using long form throughout?

Yeah, that's a good idea. It's doesn't even save any syllables. Done!

**Review by plaindocs — APPROVED** [review]

**Comment by plaindocs** [comment]
Thanks! :-)

**Comment by ericholscher** [comment]
🎉

**Pull request merged**